scripts/github_actions/generate_build_matrix.py

```diff
@@ -281,10 +281,19 @@ def generate_build_matrix(
                 else ["11.8.0", "12.1.0", "12.4.0"]
             ),
         },
+        "2.5.1": {
+            # Only Linux supports python 3.13
+            "python-version": ["3.9", "3.10", "3.11", "3.12", "3.13"],
+            "cuda": (
+                ["11.8", "12.1", "12.4"]  # default 12.4
+                if not for_windows
+                else ["11.8.0", "12.1.0", "12.4.0"]
+            ),
+        },
         # https://github.com/Jimver/cuda-toolkit/blob/master/src/links/windows-links.ts
     }
     if test_only_latest_torch:
-        latest = "2.5.0"
+        latest = "2.5.1"
```

Comment:
Please also update the if statement at line 299 to test for `2.5.1`.

Reply:
I confused.but I appended a new condition.Are these changes correct?

```
+ if "2.5.1" in matrix:
+            matrix["2.5.1"]["python-version"].remove("3.13")
+
if "2.5.0" in matrix:
            matrix["2.5.0"]["python-version"].remove("3.13")

```